### ISSUE TITLE
[Video Handler] Lightbox Video Size is Small

### ISSUE THREAD

**Issue by thebrush** [archetyped/slb-addons#28]
Hello. I have enjoyed using Simple Lightbox for a client's site 'www.PlethoraTone.com.' I purchased the video handler add-on and placed a link to a YouTube video on the home page (underneath the header and first paragraph of text. The plugin behaves wonderfully except that the lightbox that opens has the video sized at 480px wide. I am confused because the video is actually at least 1080px wide (maybe even 1920px). Shouldn't the lightbox open to display a larger version of the video? What is determining the size? I have confirmed this behavior on both PC and Mac, in Chrome and Firefox. I tested with all but Simple Lightbox (and the video add-on) de-activated to no avail. Here are the stats...

page: http://www.plethoratone.com
WP version: 4.3
SLB version: 2.5.0
Video Handler version: 0.1.0
Plugins Active:
- Haiku Minimalist Audio Player
- J Shortcodes
- Chameleon HTML5 Audio Player
- Simple Share Buttons Adder
- Superb Slideshow
- Yoast SEO

**Comment by greggluhring** [comment]
Gregg
_____________

	






On Aug 18, 2015, at 10:54 PM, thebrush <notifications@github.com> wrote:

Hello. I have enjoyed using Simple Lightbox for a client's site 'www.PlethoraTone.com.' I purchased the video handler add-on and placed a link to a YouTube video on the home page (underneath the header and first paragraph of text. The plugin behaves wonderfully except that the lightbox that opens has the video sized at 480px wide. I am confused because the video is actually at least 1080px wide (maybe even 1920px). Shouldn't the lightbox open to display a larger version of the video? What is determining the size? I have confirmed this behavior on both PC and Mac, in Chrome and Firefox. I tested with all but Simple Lightbox (and the video add-on) de-activated to no avail. Here are the stats...

page: http://www.plethoratone.com
WP version: 4.3
SLB version: 2.5.0
Video Handler version: 0.1.0
Plugins Active:

	• Haiku Minimalist Audio Player
	• J Shortcodes
	• Chameleon HTML5 Audio Player
	• Simple Share Buttons Adder
	• Superb Slideshow
	• Yoast SEO
